projects/plugins/jetpack/class.jetpack.php

```diff
@@ -1730,16 +1730,9 @@ public static function current_user_ip( $check_all_headers = false ) {
 	public static function load_modules() {
 		$status = new Status();
 
-		if ( method_exists( $status, 'is_onboarding' ) ) {
-			$is_onboarding = $status->is_onboarding();
-		} else {
-			$is_onboarding = self::is_onboarding();
```

Comment:
This seems like a good opportunity to delete the `is_onboarding()` method from this `Jetpack` class. The method is already deprecated, since 2022, and it isn't used elsewhere. Let's remove it now?

Reply:
I'm doing that in https://github.com/Automattic/jetpack/pull/39229

Reply:
https://github.com/Automattic/jetpack/pull/39229/commits/e7afbb3856e34ae6d07d10db79c01995f6f70acd